@types/pal/audio.d.ts

```diff
@@ -0,0 +1,121 @@
+declare module 'pal/audio' {
+    export interface OneShotAudio {
+        /**
+         * stop playing one shot
+         */
+        stop ();
+        /**
+         * register the play finish callback
+         * @param cb
+         */
+        onPlay (cb): OneShotAudio;
+        /**
+         * register the end callback
+         * @param cb
+         */
+        onEnded (cb): OneShotAudio;
+    }
+    export class AudioPlayer {
+        private constructor (nativeAudio: unknown);
+        /**
+         * destroy AudioPlayer
+         */
+        destroy ();
+        /**
+         * load AudioPlayer
+         * @param url
+         * @param opts
+         */
+        static load (url: string, opts?: import('pal/audio/type').AudioLoadOptions): Promise<AudioPlayer>;
+        /**
+         * load native audio for playing one shot
+         * @param url
+         * @param opts
+         */
+        static loadNative (url: string, opts?: import('pal/audio/type').AudioLoadOptions): Promise<any>;
+        /**
+         * max audio channel, if the amount of playing audios exceeds maxAudioChannel, some audio instances should be discarded by audio manager
+         */
+        static maxAudioChannel: number;
+
+        /**
+         * type of AudioPlayer，there are WEB_AUDIO and DOM_AUDIO for web platform
+         */
+        get type (): import('pal/audio/type').AudioType;
+        /**
+         * state of AudioPlayer，restores to INIT when the audio ended playing
+         */
+        get state (): import('pal/audio/type').AudioState;
+        /**
+         * whether to loop
+         */
+        get loop (): boolean;
+        set loop (val: boolean);
+        /**
+         * volume of AudioPlayer
+         */
+        get volume (): number;
```

Comment:
Does the volume can be set in any range, or [0, 1] / [0, 100]? Please clarify the range.